### ISSUE TITLE
Need help understanding the training result!

### ISSUE THREAD

**Issue by matonglidewazi** [Unity-Technologies/ml-agents#148]
Hi,

Thanks for this platform, it is awesome, compatible with almost everything! 

I'm  a conducting experiment with OpenBCI (Brain wave capture device),

I just connected the devices and solved communication between it and Unity, now we can use the brainwave as the input state, the program runs smoothly, 

and our action is the Color, HSV stimulating the eyes of people,

Currently I've done small demos, lasting for around 4 minutes, 8 minutes and 15 minutes,

the results are as follow:

![ask](https://user-images.githubusercontent.com/22267548/33012662-9106ca1a-ce1c-11e7-9b63-2b894317c45e.JPG)

The problem is, although the reward is climbing (the alpha wave stable, which is our target), the entropy 
decline very little. **I'm having trouble understanding the entropy here**. In a classification task, the cross entropy could be regard as a benchmark of the model precision, but what does entropy means in PPO?

And since it's a human experiment, our device is not so light and the tester has to face the screen. In a word, the time span for one experiment is limited to no more than 20 mins, or the tester would behave abnormaly. Is there any method to accelerate this process?

Thanks for your time,

**Comment by awjuliani** [comment]
Hi @matonglidewazi,

This is very cool work that you are doing, and I am happy to provide support on this!

In this case, entropy refers to the "randomness" in the policy. If you are using discrete actions, then you can expect this to decrease over time toward 0, which would correspond to the probability distribution over actions all focusing on a single action. When using discrete control, the `beta` parameter controls the strength of the entropy regularization, which encourages exploration. However, if `beta` is too large, then the policy will remain random and never learn specific actions. The opposite can happen where the policy quickly becomes deterministic in a "local minima." If you are using continuous actions however, then the entropy is learned, and there is no `beta` to adjust.

Using reinforcement learning with human data is a unique challenge. As you mention, with participants which can only provide 20 mins of data at a time, there is only so much which can be collected. If the data you have now isn't sufficient, I would recommend exploring off-policy RL methods (Such as DQN for discrete control, or DDPG for continuous control), which could be used to gather data from many different participants, and then collected together to learn a single policy.

**Comment by matonglidewazi** [comment]
Hi @awjuliani 

Yeah thanks for your help!

In this case we are using continous actions, but we don't know when to stop. 

Indeed at start the alpha wave could only be stable for 5s, 10 min later this number goes up to like 150s, while the entropy is still there, drop from 4.26 to 4.25. So how could we assess this, I mean, is that a insufficient training? Is that a effective model that could be used to mollify people at that stage?

**Comment by awjuliani** [comment]
@matonglidewazi,

Since it is continuous control, the entropy parameter isn't of such importance. I would focus on the reward signal. Typically in RL problems there is some "optimal" policy which provides the maximum reward. In your case, that would correspond to maximizing time where alpha-wave signal is being picked up, correct?

**Comment by matonglidewazi** [comment]
@awjuliani 

Ohh, if that's the case, then there is nothing to worry for me,

You're right ,we should focus on the reward, and I was just wondering if that's a local optimization or an overall version, 

Thank you so much!

**Comment by matonglidewazi** [comment]
![553](https://user-images.githubusercontent.com/22267548/33108750-ba5e02e0-cf78-11e7-8b4d-3056f6efe9b8.JPG)
Just train a new model with 500 thousand steps, seems to reach a platform.

**Comment by awjuliani** [comment]
@matonglidewazi Does that correspond with behavior of the "agent" which seems desirable? The algorithm is also designed to linearly decrease the learning rate, so there should naturally be smaller changes toward the end of the training round.

**Comment by matonglidewazi** [comment]
@awjuliani 

Hopefully desireable, so we could regard it as a successful trained model with 500k step?

Is that the idea?

Thank you,
